ni/heroku.py

```diff
@@ -18,6 +18,10 @@ def contrib_auth_token():
         return os.environ['GH_AUTH_TOKEN']
 
     @staticmethod
+    def contrib_secret_token():
+        return os.environ['GH_SECRET_TOKEN']
```

Comment:
If this comes back empty or the key is missing, log that fact to help in debugging when the key is forgotten.